pymarc/record.py

```diff
@@ -444,6 +444,27 @@ def isbn(self):
             pass
         return None
 
+
+    def lcc_controlnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Control Number of the record. 
+        """
+        
+        if self['010']:
+            return self['010'].format_field()
+        return None
+        
+    def lcc_callnumber(self):
+        """
+        Returns the Library of Congress Classification (LCC)
+        Call Number of the record.
+        """
+        
+        if self['050']:
```

Comment:
We should definitely check for multiples, I'm sorry I didn't think about that. Also I don't entirely follow what you mean by indicators for showing it is assigned by LC. Could you elaborate on that.